.github/workflows/abi.yaml

```diff
@@ -89,7 +89,8 @@ jobs:
 
     - name: Run tests
       run: |
-        TEST_IMAGE="postgres:${{ matrix.tester }}-alpine"
+        # Use fixed alpine version , so that libssl version does not change
```

Comment:
If you don't know the context, the comment might be hard to understand (why should the libssl version not change?). Could you describe this in a little more detail in the comment?

Reply:
Changed